helpers/search_helper.rb

```
@@ -448,6 +448,143 @@ def validate_params_solr_population(allowed_includes_params)
+        end
+
+        unless params['sort']
+          if !text.nil? && text[-1] == '*'
+            docs.sort! { |a, b| [b[:score], a[:prefLabelExact].downcase, b[:ontology_rank]] <=> [a[:score], b[:prefLabelExact].downcase, a[:ontology_rank]] }
+          else
+            docs.sort! { |a, b| [b[:score], b[:ontology_rank]] <=> [a[:score], a[:ontology_rank]] }
+          end
+        end
+
+        page_object(docs, total_found)
+      end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for process_search is too high. [<26, 70, 15> 76.16/23]

(Metrics/AbcSize)